build-support/bin/generate_travis_yml.py

```diff
@@ -551,44 +551,46 @@ def unit_tests(python_version: PythonVersion) -> Dict:
 
 def _build_wheels_command() -> List[str]:
     return [
-        "./build-support/bin/check_pants_pex_abi.py abi3 cp36m",
-        "RUN_PANTS_FROM_PEX=1 ./build-support/bin/release.sh -n",
+        # We delete the Pex's copy of `native_engine.so` (see get_ci_bootstrapped_pants_pex.sh)
+        # because the Python version used for the Pex might be different than what we use to build
+        # the wheels.
+        "rm src/python/pants/engine/internals/native_engine.so",
```

Comment:
I don't think the rm is needed. The .so is regenerated for a python version change:
https://github.com/pantsbuild/pants/blob/45449fedf237cea8adad00c28d33cf28948cfc2f/build-support/bin/native/calculate_engine_hash.sh#L22-L35

Reply:
True, good point. I will remove that in a followup as I'll be touching this code later today.